### PR TITLE
Tvs/weather layers fix

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
The weather layers aren't working.

The date fns times are zoned times where as the old api calls were made with utc time. I

I tried using date-fns-tz to convert to utc but this stopped the app from loading on my environment.

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
I really hate time zones.

Guess I'll give it a 3rd shot in a few months...

**Comment by VIKTORVAV99** [comment]
Looks like you need to update the yarn lockfile as well.

**Comment by tonypls** [comment]
Yeah timezones are a pain, moment has a small place in my heart <3

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (VIKTORVAV99) [inline yarn.lock]: I'm glad it's not you me that accidentally run yarn on the top level. 😅

**Pull request merged**